src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -84,29 +69,37 @@ const ProjectExternalSubmissionsTable = ({
       {
         header: 'Date Submitted',
         linkTreatment: false,
-        render: (s: ExternalFormSubmissionFieldsFragment) =>
+        render: (s: ExternalFormSubmissionSummaryFragment) =>
           parseAndFormatDateTime(s.submittedAt),
       },
-      ...customColumns,
     ];
   }, []);
 
-  const [selected, setSelected] =
-    useState<ExternalFormSubmissionFieldsFragment | null>(null);
+  const [selectedId, setSelectedId] = useState<string | null>(null);
+
+  const { data: { externalFormSubmission: selected } = {}, error } =
+    useGetExternalFormSubmissionQuery({
+      variables: {
+        id: selectedId || '',
+      },
+      skip: !selectedId,
+    });
 
   const submissionValues = useMemo(() => {
-    if (!selected || !definition) return {};
-    const itemMap = getItemMap(definition.definition, true);
+    if (!selected) return {};
+    const itemMap = getItemMap(selected.definition.definition, true);
     const submissionValues: FormValues = {};
     Object.keys(itemMap).forEach((key) => {
       const item = itemMap[key];
       const customFieldKey = item.mapping?.customFieldKey;
-      if (!customFieldKey) return;
 
-      const value = customDataElementValueForKey(
-        customFieldKey,
-        selected.customDataElements
-      );
+      // TODO - this doesn't support reading the record type from the parent's record type. see transformSubmitValues
```

Comment:
Let's discuss this -- I think the impact is that with what I have currently, external forms don't support setting a record type on a Group and then having the children of that group use that record type without needing to individually specify. It looks(?) like regular forms do support that. Do we use it anywhere? Do we plan to on the PIT?

Reply:
@martha I think that was previous behavior (before we introduced the `mapping` object) that is no longer needed. I just verified in the [schema](https://github.com/greenriver/hmis-warehouse/blob/release-132/drivers/hmis_external_apis/public/schemas/form_definition.json) that it's not valid to have a mapping on a group, so we shouldn't need to worry about that and can probably pull the `parentRecordType` code out of `transformSubmitValues`.

Reply:
Sounds good thanks! I removed the todo. Should I pull that out of `transformSubmitValues` on this PR or separately?

Reply:
I think a separate PR would be wise. THanks!